site/realm/data_sources/mongodb-atlas/history/incidents/schema.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
I would like to reuse the same schema since there is a significant chance of them going out of sync.

Maybe we could write a test that compares both and checks that they are the same except for the particular fields here 🤔

Reply:
Added!